x-pack/legacy/plugins/infra/common/inventory_models/host/detail_page.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FunctionComponent, useEffect, useMemo } from 'react';
+import { EuiSideNav, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { InfraMetricData } from '../../../public/graphql/types';
+import { Section } from '../../../public/pages/metrics/components/section';
+import { SubSection } from '../../../public/pages/metrics/components/sub_section';
+import { NavItem, SideNavContext } from '../../../public/pages/metrics/lib/side_nav_context';
+
+interface VisSectionProps {
+  metric?: InfraMetricData;
+}
+
+const VisSection: FunctionComponent<VisSectionProps> = ({ metric }) => {
+  if (metric) {
+    return (
+      <div
+        style={{
+          display: 'flex',
+          width: '100%',
+          height: 250,
+          backgroundColor: '#CCC',
+          alignItems: 'center',
+        }}
+      >
+        <div style={{ textAlign: 'center', flex: '1 0 auto' }}>VISUALIZATION GOES HERE</div>
+      </div>
+    );
+  }
+  return <div>Gauges without Metric (you should never see this)</div>;
+};
+
+interface WithMetricsDataProps {
+  source: string;
+  ids: string[];
+  children: (props: LayoutProps) => React.ReactElement;
+}
+
+const WithMetricsData = ({ ids, children }: WithMetricsDataProps) => {
+  if (!children) {
+    return null;
+  }
+
+  // eslint-disable-next-line no-console
+  useEffect(() => console.log('Fetching', ids), [ids]);
```

Comment:
Just want to make sure this is intentional

Reply:
Actually that whole file needs to be removed. That was just the prototype.